docs/sources/setup-grafana/installation/docker/index.md

```diff
@@ -8,253 +8,313 @@ weight: 200
 
-**Grafana Open Source edition**: `grafana/grafana-oss:<version>`
+```bash
+docker run -d -p 3000:3000 --name=grafana grafana/grafana-enterprise
+```
 
-The default images are based on the popular [Alpine Linux project](http://alpinelinux.org), available in [the Alpine official image](https://hub.docker.com/_/alpine). Alpine Linux is much smaller than most distribution base images, and thus leads to slimmer and more secure images.
+where:
 
-The Alpine variant is highly recommended when security and final image size being as small as possible is desired. The main caveat to note is that it uses [musl libc](http://www.musl-libc.org) instead of [glibc and friends](http://www.etalabs.net/compare_libcs.html), so certain software might run into issues depending on the depth of their libc requirements. However, most software don't have an issue with this, so this variant is usually a very safe choice.
+run = run directly from the command line\
+d = run in the background\
+p = assign the port number, which in this case is `3000`\
+name = assign a logical name to the container, for example, `grafana`\
+grafana/grafana-enterprise = the image to run in the container
 
-> **Note:** Grafana docker images were based on [Ubuntu](https://ubuntu.com/) prior to version 6.4.0.
+### Stop the Grafana container
 
-## Ubuntu image
+To stop the Grafana container, run the following command:
 
-**Grafana Enterprise edition**: `grafana/grafana-enterprise:<version>-ubuntu`
+```bash
+# docker ps command tells about the running process in the docker
+docker ps
 
-**Grafana Open Source edition**: `grafana/grafana-oss:<version>-ubuntu`
+# This will display a list of containers, like:
+CONTAINER ID   IMAGE  COMMAND   CREATED  STATUS   PORTS    NAMES
+cd48d3994968   grafana/grafana-enterprise   "/run.sh"   8 seconds ago   Up 7 seconds   0.0.0.0:3000->3000/tcp   grafana
 
-These images are based on [Ubuntu](https://ubuntu.com/), available in [the Ubuntu official image](https://hub.docker.com/_/ubuntu). It is an alternative image for those who prefer an [Ubuntu](https://ubuntu.com/) based image and/or are dependent on certain tooling not available for Alpine.
+# To stop the grafana container run the command
+# docker stop CONTAINER-ID or use 
+# docker stop NAME (which is grafana as we defined previously)
+docker stop grafana
+```
 
-## Run Grafana
+### Save your Grafana data
 
-You can run the latest Grafana version, run a specific version, or run an unstable version based on the main branch of the [grafana/grafana GitHub repository](https://github.com/grafana/grafana).
+When you use Docker containers, the data inside them is temporary by default. This means that if you don't specify where to store the information, all the Grafana data will be lost when you stop the Docker container. To avoid losing your data, you can set up [persistent storage](https://docs.docker.com/storage/volumes/) or [bind mounts](https://docs.docker.com/storage/bind-mounts/) for your container.
```

Comment:
```suggestion
When you use Docker containers, their data is temporary by default. If you don't specify where to store the information, all the Grafana data will be lost when you stop the Docker container. To avoid losing your data, you can set up [persistent storage](https://docs.docker.com/storage/volumes/) or [bind mounts](https://docs.docker.com/storage/bind-mounts/) for your container.
```